widget/popup.go

```diff
@@ -53,15 +53,24 @@ func (p *PopUp) Resize(size fyne.Size) {
 	p.Refresh()
 }
 
-// Show this widget, if it was previously hidden
+// Show this pop-up as overlay if not already shown.
 func (p *PopUp) Show() {
 	if !p.overlayShown {
+		if (p.Size() == fyne.Size{}) {
```

Comment:
nit: unnecessary parenthesis

Reply:
No, the `fyne.Size{}` requires the parenthesis here.

Reply:
Ah ok, creating a new object to compare with Size to see if it is 0,0 is not very elegant. Maybe Size should have an "isEmpty" or "isZero" method?

Reply:
I would assume that the Go compiler is able to optimise comparisons with zero values by only allocating the needed value once (or at least not that often) or by applying an optimised zero value test procedure. Of course it might not do such stuff.

Before adding a very special function for a limited purpose I want to see benchmarks that prove its benefits.
Is calling a function which compares two `int` values really faster than the current comparison?

Reply:
#849

Reply:
Do you agree that the function isn’t significantly faster?

Reply:
Hahaha you didn't ask for a benchmark showing it is significantly faster :)

As far as execution time goes 10ns is nothing to a human, and a handful of opcodes to a CPU.

As far as code goes
```
if p.Size().Empty() {
```
is significantly more elegant and readable than
```
if (p.Size() == fyne.Size{}) {
```

Reply:
I don’t really have an opinion on this, but if a method were added I imagine that Zero() fits the requirement more than Empty()

Reply:
I just introduced `IsZero()`…

Reply:
Thanks, it is much more readable 👍